src/pages/en/guides/styling.md

```diff
@@ -4,14 +4,23 @@ title: Styling & CSS
 description: Learn how to style components with Astro.
 ---
 
-Astro includes special handling to make writing CSS as easy as possible. Styling inside of Astro components is done by adding a `<style>` tag anywhere.
 
-## Astro component styles
+INTRO BLAH BLAH: Astro includes special handling to make writing CSS as easy as possible. You can use your favorite libraries, write your own css files, or take advantage of Astro's own scoped styling to target UI frameworks at the component level.
 
-By default, all Astro component styles are **scoped**, meaning they only apply to the current component. This can be very easy to work with, as you only have to worry about what’s in your current document at any given time.
+Astro automatically minifies and bundles...
 
-```html
-<!-- src/components/MyComponent.astro -->
+## Astro scoped `<style>`
+
+Styling inside of Astro components is done by adding a `<style>` tag in the component template.
+
+By default, all Astro component styles are **scoped**, meaning they only apply to the HTML elements written in the current component. Any HTML rendered via a child component import is **not** affected by the `<style>` tag.
```

Comment:
It’s probably OK to leave this like this, but this is not 100% accurate because of how CSS cascades/inherits. Child components won’t be targeted directly by the selectors in `<style>`, but something like the following example would still see the child component inherit a smaller font size.

```astro
<div class="small">
  Small text
  <Component />
</div>

<style>
.small { font-size: 0.5em; }
</style>
```

That said, I can also see the value of the current version for someone who is like “Hey, I _said_ H1s should be red, dammit, why is my child component not red?!”

Reply:
I *do* want to be accurate! At the same time, we get a lot of "Why isn't my h1 red?!?!"  So, let's think of wording/an example that satisfies both!